test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,32 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: Application, DB and Service Binding Request
+        Given Imported Nodejs application "nodejs-rest-http-crud-a-d-s" is running
+        * DB "db-demo-a-d-s" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-a-d-s
+            spec:
+                applicationSelector:
+                    resourceRef: nodejs-rest-http-crud-a-d-s
+                    group: apps
+                    version: v1
+                    resource: deployments
+                backingServiceSelector:
+                    group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    resourceRef: db-demo-a-d-s
+                    id: postgresDB
+                customEnvVar:
+                    - name: tag
```

Comment:
we bind here only a sequence, right? Perhaps we should name the scenario like that that. We still then need a scenario where we bind a map, it seems?

Reply:
The PR is in draft. Not ready for review. :)